packages/demo/index.tsx

```diff
@@ -98,6 +98,14 @@ const App = () => {
 
       <hr />
 
+      <div>
+        <h2>Connect identity</h2>
```

Comment:
Regarding adding the ability to change the connected identity, this change wont resolve it. Because that is another feature which is not supported yet its related to this: 
- #888 
- #945

Reply:
That is also related to the `Connected` and `Not Connected` word shown in the extension UI. Which in the current implementation it change to `Connected` if the user Approve but not necessary if the user choose an identity to connect with. Which means the user could approve from the approval screen and then could reject the next screen of choosing an identity, but at the same time you would see `Connected` which is confusing ofc for the user (technically and logically the connection didn't happen yet). 
That all should be resolved with another feature of having a separate connection service.

Reply:
With this change I can change identities for demo without reinstalling the extension.